libfreerdp/core/fastpath.h

```diff
@@ -91,7 +91,9 @@ enum FASTPATH_INPUT_EVENT_CODE
 	FASTPATH_INPUT_EVENT_MOUSE = 0x1,
 	FASTPATH_INPUT_EVENT_MOUSEX = 0x2,
 	FASTPATH_INPUT_EVENT_SYNC = 0x3,
-	FASTPATH_INPUT_EVENT_UNICODE = 0x4
+	FASTPATH_INPUT_EVENT_UNICODE = 0x4,
+	TS_FP_RELPOINTER_EVENT = 0x5,
+	TS_FP_QOETIMESTAMP_EVENT = 0x6
```

Comment:
Nit: `2.2.8.1.2.2 Fast-Path Input Event (TS_FP_INPUT_EVENT)` defines these name-wise as `FASTPATH_INPUT_EVENT_MOUSEREL` and `FASTPATH_INPUT_EVENT_QOE_TIMESTAMP`.

Or was the intention here a shorter name? I think we should keep the convention consistent here (I'm fine with a further API break (we already break in FreeRDP3)).

Reply:
No, the event codes all start with `FASTPATH_INPUT_EVENT_`. See also: https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpbcgr/76c4dd59-7ba0-445d-a03c-885212ab80f6

Reply:
`TS_FP_INPUT_EVENT` is just the name of the fastpath PDU in general